acoustics/acoustics_interface/acoustics_interface/acoustics_interface_lib.py

```diff
@@ -158,22 +219,30 @@ def get_ip(cls):
         return IP
 
     @classmethod
-    def send_acknowledge_signal(cls):        
+    def _send_acknowledge_signal(cls) -> None:   
+        """
+            Sends "HELLO :D to teensy
+        """     
         try:
-            cls.clientSocket.sendto(cls.INITIALIZATION_MESSAGE.encode(), cls.address)
+            cls._clientSocket.sendto(cls._INITIALIZATION_MESSAGE.encode(), cls._address)
             print("DEBUGING: Sent acknowledge package")
         except Exception as e:
             print("Error from send_acknowledge_signal")
             print(e)
             pass
 
     @classmethod
-    def check_if_available(cls):
+    def _check_if_available(cls) -> None:
+        """
+            Checks if READY has been received
+
+            Note: The while loop here may not be necessary
```

Comment:
explain here why, that is 99% cases this while loop is not necesarry, but in case we miss a case, then we have it for redundancy

Reply:
Fixed :)